packages/mobx/src/core/computedvalue.ts

```diff
@@ -90,8 +100,13 @@ export class ComputedValue<T> implements IObservable, IComputedValue<T>, IDeriva
     protected value_: T | undefined | CaughtException = new CaughtException(null)
     name_: string
     triggeredBy_?: string
-    isComputing_: boolean = false // to check for cycles
-    isRunningSetter_: boolean = false
+
+    private static readonly isComputingIndex = 0
+    private static readonly isRunningSetterIndex = 1
+    private static readonly isBeingObservedIndex = 2
+    private static readonly isPendingUnobservationIndex = 3
```

Comment:
if we use value 0 1 2 4 directly, does it make it slightly faster by skipping the bit shifting?

Reply:
Done! Yes I think that's neater thanks